client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -432,6 +437,61 @@ function recordOrderInGoogleAds( cart, orderId ) {
 	}
 }
 
+function recordOrderInGAEnhancedEcommerce( cart, orderId, wpcomJetpackCartInfo ) {
+	if ( ! isAdTrackingAllowed() ) {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] ad tracking is disallowed' );
+		return;
+	}
+
+	if ( ! isGoogleAnalyticsEnabled || ! isGoogleAnalyticsEnhancedEcommerceEnabled ) {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] Google Analytics is not enabled' );
+		return;
+	}
+
+	let products, brand, totalCost;
+
+	if ( wpcomJetpackCartInfo.containsWpcomProducts ) {
+		products = wpcomJetpackCartInfo.wpcomProducts;
+		brand = GA_PRODUCT_BRAND_WPCOM;
+		totalCost = wpcomJetpackCartInfo.wpcomCost;
+	} else if ( wpcomJetpackCartInfo.containsJetpackProducts ) {
+		products = wpcomJetpackCartInfo.jetpackProducts;
+		brand = GA_PRODUCT_BRAND_JETPACK;
+		totalCost = wpcomJetpackCartInfo.jetpackCost;
+	} else {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] No products' );
+		return;
+	}
+
+	const items = [];
+	products.map( product => {
+		items.push( {
+			id: product.product_id,
+			name: product.product_name_en,
+			quantity: product.volume,
+			price: product.cost,
+			brand,
+		} );
+	} );
+
+	const params = [
+		'event',
+		'purchase',
+		{
+			transaction_id: orderId,
+			value: totalCost,
+			currency: cart.currency ?? 'USD',
```

Comment:
Let's do all USD so the total in GA across the board are easier to deal with. This also aligns with the approach we've taken in other 3rd-party trackers.